scripts/prepare-icons.js

```diff
@@ -34,6 +34,9 @@ const SVGO_CONFIG = {
     generateIconNameType(
       icons.map((_) => basename(_, SVG_EXT)),
       processedIcons.map(([title]) => title),
+      processedIcons.map(([title, svg, inverse]) => {
```

Comment:
Если не будет падать, то можно и так:
```suggestion
      processedIcons.map(([_, _, inverse]) => {
```

Reply:
Код будет падать, оставить как есть